resteasy-jaxrs/src/main/java/org/jboss/resteasy/plugins/delegates/ClientCookieHeaderDelegate.java

```diff
@@ -0,0 +1,29 @@
+package org.jboss.resteasy.plugins.delegates;
+
+import javax.ws.rs.core.NewCookie;
+
+import org.jboss.resteasy.resteasy_jaxrs.i18n.Messages;
+
+/**
+ * 
+ * @author <a href="mailto:ron.sigal@jboss.com">Ron Sigal</a>
+ * @date May 20, 2016
+ */
+public class ClientCookieHeaderDelegate extends NewCookieHeaderDelegate
+{
+   public String toString(Object value)
+   {
+      if (value == null)
+      {
+         throw new IllegalArgumentException(Messages.MESSAGES.paramNull());
+      }
+      NewCookie cookie = (NewCookie) value;
+      StringBuilder b = new StringBuilder();
+      b.append(cookie.getName()).append('=');
+      if (cookie.getValue() != null)
+      {
+         quote(b, cookie.getValue());
```

Comment:
Hey Thomas,

Thank you for the clarification. Ok, I'm surrendering to RFC 2109. I've backed out some of my changes and updated CookieHeaderDelegate to print $Version, $Path, and $Domain, and it looks like that doesn't break anything. I'm going to close this pull request, as it is superceded by #827. And I'll mention JAX_RS_SPEC-430 to our representative to the JAX-RS expert group.

-Ron